.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ jobs:
             python3 -mvenv /usr/local/share/virtualenvs/tap-pendo
             source /usr/local/share/virtualenvs/tap-pendo/bin/activate
             pip install -U pip setuptools
-            pip install .[dev]
+            pip install .[test]
```

Comment:
@prijendev Does this indicate that we no more consider the modules added as part of the 'dev' list?